datasets/medmcqa/README.md

```diff
@@ -0,0 +1,231 @@
+    "exp": "Circulating levels of leukocytes and their precursors may occasionally reach very high levels (>50,000 WBC mL). These extreme elevations are sometimes called leukemoid reactions because they are similar to the white cell counts observed in leukemia, from which they must be distinguished. The leukocytosis occurs initially because of the accelerated release of granulocytes from the bone marrow (caused by cytokines, including TNF and IL-1) There is a rise in the number of both mature and immature neutrophils in the blood, referred to as a shift to the left. In contrast to bacterial infections, viral infections (including infectious mononucleosis) are characterized by lymphocytosis Parasitic infestations and certain allergic reactions cause eosinophilia, an increase in the number of circulating eosinophils. Leukopenia is defined as an absolute decrease in the circulating WBC count.",
+    "cop":1,
+    "opa":"Leukemoid reaction",
+    "opb":"Leukopenia",
+    "opc":"Myeloid metaplasia",
+    "opd":"Neutrophilia",
+    "subject_name":"Pathology",
+    "topic_name":"Basic Concepts and Vascular changes of Acute Inflammation",
+    "id":"4e1715fe-0bc3-494e-b6eb-2d4617245aef",
+    "choice_type":"single"
+}
+```
+### Data Fields
+
+- `id` : a string question identifier for each example
+- `question` : question text (a string)
+- `opa` : Option A
+- `opb` : Option B
+- `opc` : Option C
+- `opd` : Option D
+- `cop` : Correct option, i.e., 1,2,3,4
+- `choice_type` : Question is `single-choice` or `multi-choice`
+- `exp` : Expert's explanation of the answer
+- `subject_name` : Medical Subject name of the particular question
+- `topic_name` : Medical topic name from the particular subject
+
+### Data Splits
+
+The goal of MedMCQA is to emulate the rigor of real word medical exams. To enable that, a predefined split of the dataset is provided. The split is by exams instead of the given questions. This also ensures the reusability and generalization ability of the models. 
+
+The training set of MedMCQA consists of all the collected mock & online test series, whereas the test set consists of all AIIMS PG exam MCQs (years 1991-present). The development set consists of NEET PG exam MCQs (years 2001-present) to approximate real exam evaluation.
+
+Similar questions from train , test and dev set were removed based on similarity. The final split sizes are as follow:
+
+|                             | Train   | Valid | Test |
+| -----                       | ------ | ----- | ---- |
+| Question #| 182,822 |  6,150 | 4,183|
+| Vocab       | 94,231 |  11,218 | 10,800 |
+| Max Ques tokens    | 220  |  135| 88 |
+| Max Ans tokens | 38 | 21 | 25 |
```

Comment:
There is a mistake with this table (and also in the table appearing in the GitHub repo): the validation and test columns are interchanged.
- The validation split has 4,183 questions (the same applies to the rest of the column values)
- The test split has 6,150 questions

This information is correct in the paper (Table 2).

Reply:
Ok, I've changed the column names.